### PR TITLE
Us004 view band level for each role

### PR DESCRIPTION
**Pull request by BartlomiejMilewski**
### 🛠 Changes being made

Here give examples of the changes you've made in this pull request. Include an itemized list if you can. It'll help the reviewer

### ✨ Definition of Done (all the points need to be filled out before merging)

- [ ] Meets Acceptance Criteria
- [ ] Tested
- [ ] Code Review made by team 2 members
- [ ] UX approved
- [ ] Product Owner Approved

### 🧪 Non-Functional Requirements

- [ ] Automated tests should cover acceptance criterias (Unit tests, Integration/API/Component tests)
- [ ] All input must be validated (client side & API)
- [ ] Sensitive data should not be stored in the URL
- [ ] No secrets in codesource

### CODE REVIEW

File: product-backend/src/main/java/org/kainos/ea/db/JobRoleDao.java [hunk 1]
```diff
@@ -18,24 +19,29 @@ public JobRoleDao() {
         databaseConnector = new DatabaseConnector();
     }
 
-    public List<JobRole> getAllJobRoles() throws SQLException, DatabaseConnectionException {
+    public List<JobRolePlusBandResponse> getAllJobRoles() throws SQLException, DatabaseConnectionException {
```
Comment (Fluffyguy611) [inline product-backend/src/main/java/org/kainos/ea/db/JobRoleDao.java]: Could change to "JobRoleWithBand"

**Review by Fluffyguy611 — APPROVED** [review]

**Review by mtslech — APPROVED** [review]
Ok

File: product-web-app/views/view-single-jobRole.html [hunk 2]
```diff
@@ -86,8 +86,8 @@
                         <td class="role-details-td">{{ jobRole.description }} </td>
                         <td class="role-details-td"><a href="{{ jobRole.sharePointLink }}">
                             <button class="details-job-role">View Details</button>
-                        <td class="role-details-td">{{ Band.name }} </td>
-                        <td class="role-details-td">{{ Band.level }} </td>
+                        <td class="role-details-td">{{ jobRole.bandName }} </td>
```
Comment (ljercha) [inline product-web-app/views/view-single-jobRole.html]: Display it below like this BandName(BandLevel)
do not add additional column

File: product-backend/src/main/java/org/kainos/ea/db/JobRoleDao.java (hunk 1 above)
Comment (ljercha) [inline product-backend/src/main/java/org/kainos/ea/db/JobRoleDao.java]: Reuse existing JobRole model like you did on a frontend

**Review by ljercha — CHANGES_REQUESTED** [review]

**Review by ljercha — APPROVED** [review]

**Pull request merged**